File: mvp.admin/src/FlexForm.Admin/flexform-admin/src/app/models/dtos/item-source-definition.model.ts
import { BaseWithTimeStamp } from "./base-with-timestamp.model";

export class ItemSourceDefinition extends BaseWithTimeStamp<number>{
    name: string;
    itemSourceEntityDefinitionId:number;
    isSingleResult:boolean;
    itemSourceCondition:string;
    itemSourceDefaultOrder:string;
    includeProperties:string;
    pageSize:number;    
}

File: mvp.admin/src/FlexForm.Api.Admin/Model/ItemSourceDefinitionModel.cs
namespace FlexForm.Api.Admin.Model
{
    public class ItemSourceDefinitionModel : ModelBaseInt
    {
        public string Name { get; set; }

        public int ItemSourceEntityDefinitionId { get; set; }

        public bool IsSingleResult { get; set; }

        public string ItemSourceCondition { get; set; }

        public string ItemSourceDefaultOrder { get; set; }

        public string IncludeProperties { get; set; }

        public int PageSize { get; set; }
    }
}
